configs/components/echam/echam.yaml

```diff
@@ -382,26 +382,41 @@ choose_scenario:
                         swflux: histswflux
                         MAC-SP: MAC-SP
 
-        # Shared Socioeconomic Pathways (SSPs) scenarios. Forcing data are given
-        # under the choose_scenario_type block
-        ssp585:
-                scenario_type: cmip6
-
-        ssp370:
-                scenario_type: cmip6
-
-        ssp126:
-                scenario_type: cmip6
-
-        ssp245:
-                scenario_type: cmip6
-
-        ssp534os:
-                scenario_type: cmip6
-
         cmip6hist:
                 scenario_type: cmip6
 
+# Shared Socioeconomic Pathways (SSPs) scenarios. Forcing data are given
+# under the choose_scenario_type block
+scenario_is_ssp: "$(( 'ssp' in '${echam.scenario}' ))"
+choose_scenario_is_ssp:
+    True:
+        scenario_type: cmip6
+        data_set_not_r0008: "$(( '${dataset}' != 'r0008' ))"
+        choose_data_set_not_r0008:
+            True:
+                add_warning:
+                    "echam dataset":
+                        message: "
+                            You are trying to run a ``ssp`` simulation with the
+                            ``echam.dataset: ${dataset}``. Some input files might be
+                            missing for ssp simulations in that dataset. It's
+                            recommended that you use the ``r0008`` dataset instead.\n\n
+                            If you wish to continue with the current dataset answer
+                            ``Y`` to the question. If you want to switch to
+                            ``r0008`` dataset, answer ``n`` and modify your runscript
+                            accordingly (``echam.dataset: r0008``)"
+                        ask_user_to_continue: True
```

Comment:
While this works, what about others? Dataset 9, 10, 11 (doesn't exist yet, but who knows). I would recommend rather than doing a simple string comparison, we do some kind of greater-than check, but that might mean digging into the parser, which I do not what to recommend.

Reply:
On the other hand this is an error message very specific to our current situation, so generalizing might not be a good idea, at least for now. One could always generalize it like this: `data_set_lower_than_r0008: "$(( int('${dataset}'[-4:]) < 8 ))"`, but I don't think we really need that right now.

Reply:
It is nice but actually not needed since missing file checks are performed anyway?

Reply:
Agreed... I'll remove it